expression/constant.go

```diff
@@ -60,6 +60,9 @@ type Constant struct {
 	// It's only used to reference a user variable provided in the `EXECUTE` statement or `COM_EXECUTE` binary protocol.
 	ParamMarker *ParamMarker
 	hashcode    []byte
+	// InOperand indicates whether this column is the inner operand of column equal condition converted
+	// from `[not] in (subq)`.
+	InOperand bool
```

Comment:
Why do we need this field in `Constant`?

Reply:
```select a not in (select 1 )``` like this